src/composables/useQuorum.ts

```diff
@@ -9,7 +9,6 @@ interface QuorumProps {
   space: ExtendedSpace;
   proposal: Proposal;
   results: Results;
-  votes: Vote[];
```

Comment:
Why did you remove the votes prop is so mane components?

Reply:
We don't need them there, and we don't have votes in SpaceProposalPage no more, all logic is in SpaceProposalVotesList.

Reply:
Did you also check if any plugin is using `votes`?

Reply:
Didn't find any `votes` dependencies

Reply:
The votes prop is defined in multiple plugins. Can you add it back for plugins for now and create an issue to remove in another PR? 
<img width="299" alt="image" src="https://user-images.githubusercontent.com/51686767/225583972-e819d385-bd2f-48c6-8a94-a9053dbe7ec4.png">